src/transforms/es6/providetomodule.js

```diff
@@ -6,6 +6,22 @@ const {addRequire, isClosureClass, isControllerClass, isDirective, isGoogDefine,
 const {createAssignmentShim, createUIShim} = require('../../utils/shim');
 const {logger} = require('../../utils/logger');
 const {resolveThis} = require('../../utils/resolvethis');
+const {addOSGlobals, isOSGlobal, convertOSGlobal} = require('../../utils/opensphere');
+
+
+// on creation, read ".jscodeshift.json" file(s) from this and local folders
+(function(fs, workspace, filename){
+  fs.readdirSync(workspace).forEach((folder) => {
+    try {
+      const config = JSON.parse(fs.readFileSync(`${workspace}${folder}/${filename}`, 'utf8'));
+      if (config) {
+        addOSGlobals(config['globals']);
+      }
+    } catch (e) {
+      // no file found or couldn't read it
+    }
+  });
+})(require('fs'), '../', '.jscodeshift.json');
```

Comment:
Typically all `require` statements should be at the top of the file. Is there a reason this has to be a param for your IIFE?

Reply:
Moved!